framework/default-cmm.md

```diff
@@ -55,7 +59,10 @@ If the algorithm suite contains a [signing algorithm](algorithm-suites.md#signat
   - The key MUST be the reserved name, `aws-crypto-public-key`.
   - The value MUST be the base64-encoded public verification key.
 
-On each call to Get Encryption Materials, the default CMM MUST make a call to the key provider's [On Encrypt](keyring-interface.md#onencrypt) function.
+On each call to Get Encryption Materials,
+the default CMM MUST make a call to the keyring's [On Encrypt](keyring-interface.md#onencrypt) function.
+If the default CMM is configured with a master key provider instead,
```

Comment:
I would make it clear here that master key provider is both legacy
and not implemented in every runtime here as well.

Or, we could leave this out
and have the description of mater key providers
be the only description.
In that case, I would suggest a callout in the Overview.

Reply:
+1
I think I'd prefer the latter, but at the very least if we mention MPKs in the main spec let's make it more clear that this is referring to legacy behavior that one SHOULD NOT implement in new implementations.

Reply:
Yeah, I think I'm ok with that. If someone knows they're looking for MKP behavior, they shouldn't have any issues finding it in the "Legacy" section, and the descriptions in that section make it clear where they fit into the spec.